doc/moosedocs.py

```diff
@@ -26,4 +26,7 @@
 
 from MooseDocs import main
 if __name__ == '__main__':
+    os.chdir("..")
+    os.system("doxygen doc/content/doxygen/Doxyfile")
```

Comment:
You should probably verify that this command runs successfully by checking the return value. Print a warning or even end the build if it does not. It may also be worth considering using the [subprocess module](https://docs.python.org/3/library/subprocess.html#module-subprocess), although that's not massively important.

Reply:
Good idea - I've added a check that the doxygen command runs successfully